p2p/src/node.rs

```diff
@@ -1181,6 +1181,24 @@ mod tests {
     use iroh_rpc_types::{p2p::P2pAddr, Addr};
     use tracing_subscriber::{fmt, prelude::*, EnvFilter};
 
+    // #[cfg(loomtest)]
+    #[tokio::test]
+    async fn basic_loom_test() {
+        loom::model(|| {
+            let rt = tokio::runtime::Builder::new_current_thread()
+                .build()
+                .unwrap();
+            rt.block_on(async {
+                let test_runner = TestRunnerBuilder::new().no_bootstrap().build().await.unwrap();
+                let got_peer_id = test_runner.client.local_peer_id().await.unwrap();
+                let expect_peer_id: PeerId = "12D3KooWFma2D63TG9ToSiRsjFkoNm2tTihScTBAEdXxinYk5rwE"
```

Comment:
What is the seed that makes this a constant?

https://github.com/ceramicnetwork/rust-ceramic/blob/b41f2f8c25eec92f233f561e4ed329d44b3ac880/p2p/src/node.rs#L1372C17-L1379C26
```rs
// public key 12D3KooWFma2D63TG9ToSiRsjFkoNm2tTihScTBAEdXxinYk5rwE
Ed25519Keypair::from_bytes(&[
    76, 8, 66, 244, 198, 186, 191, 7, 108, 12, 45, 193, 111, 155, 197, 0, 2, 9, 43,
    174, 135, 222, 200, 126, 94, 73, 205, 84, 201, 4, 70, 60, 88, 110, 199, 251,
    116, 51, 223, 7, 47, 24, 92, 233, 253, 5, 82, 72, 156, 214, 211, 143, 182, 206,
    76, 207, 121, 235, 48, 31, 50, 60, 219, 157,
])
.unwrap()
```

Should we be logging a warning of some kind when this publicly known secret is used so we can accidentally use this number somewhere?